themes/gatsby-theme-deriv/src/features/pages/options/accumulator/index.tsx

```diff
@@ -29,7 +32,12 @@ const swiperOption: SwiperOption = {
     freeMode: true,
 }
 
-const AccumulatorsOptions = ({region}: BuildVariantType) => {
+const AccumulatorsOptions = ({ region }: BuildVariantType) => {
+    const [isModalOpen, setIsModalOpen] = useState({ is_open: false, video_id: '' })
+
+    const handleModalToggle = (action: boolean, video_id?: string) => {
+        action && setIsModalOpen({ is_open: !isModalOpen.is_open, video_id: video_id || '' })
+    }
```

Comment:
why do we need `action`?

Reply:
and how this `video_id` thing working?

Reply:
we need action to prevent open video modal to every cards we need only one card to be open on the click .
nly the first element has on click others doesn't 
 cards={accumulatorOptionsVideoCard.map((card) => ({
                    ...card,
                    content: (
                        <div
                            onClick={() =>
                                handleModalToggle(card.id === 1, how_to_trade_accumulator_video_id)
                            }
                        >
                            {card.content}
                        </div>
                    ),
                }))}

Reply:
i am using videoModal component only once and passing video_id as props